### DISCUSSION TITLE
There is a display bug in the mobile interface.

### DISCUSSION THREAD

**Discussion by fxzxmicah** [community/community#14233] [Other Feature Feedback, Questions, & Ideas]
![Screenshot_20220404_163237](https://user-images.githubusercontent.com/48860358/161506249-c3bf9e4d-e8b5-4d3a-b9f8-87c7a7ce1df1.jpg)
There is a blank column on the right.
The browser is Microsoft Edge.

**Comment by iifawzi** [comment]
Same here, using safari, iphone 12 
![CC342A5D-3696-4DC6-982A-3D99A6C12766](https://user-images.githubusercontent.com/46695441/161663656-e4aca824-4e6b-4af3-9539-58398de5e264.png)

**Comment by alex-kim-dev** [comment]
A long list of reactions is causing it
![github com_](https://user-images.githubusercontent.com/45559664/161811855-29802c47-3d93-4257-b750-da503bd3d0c2.png)

**Comment by Marychoffel22** [comment]
Assigned

**Comment by Marychoffel22** [comment]
Pull requests

**Comment by Marychoffel22** [comment]
full access

**Reply by fxzxmicah** [reply]
Thanks for your work. Actually, this bug is also existing in the releases page with emoji.
Like @alex-kim-dev mentioned https://github.com/github/feedback/discussions/14233#discussioncomment-2509872 .
![Screenshot_20220416_181504](https://user-images.githubusercontent.com/48860358/163671264-bd0e5e6a-20dd-4c93-a4a6-8007c9b707c1.jpg)

**Reply by alex-kim-dev** [reply]
Yep, and there's also text mentioning reactions count that makes the reactions component even longer
Screenshot (375px) from [typescript releases](https://github.com/microsoft/TypeScript/releases):

<details><summary>Screenshot</summary>

![github com_microsoft_TypeScript_releases_tag_v4 7-beta](https://user-images.githubusercontent.com/45559664/163673134-632fa614-cb7a-4bbe-8db5-bd7a2dff39c9.png)
</details>
